### ISSUE TITLE
Action not available for selection

### ISSUE THREAD

**Issue by chicagojcb** [douglascayers-org/sfdx-mass-action-scheduler#157]
## Summary
Hi, I know you dealt with this some years ago, but it is happening to me now and I don't know what to do next.  It could be operator error rather than a bug, but I can't know for sure.  

## Version of Mass Action Scheduler
2.4

## Steps To Reproduce
For starters, let me tell you that I have assumed operator error and I took the precaution of entirely deleting and uninstalling and starting over, assuming I had mis-stepped somewhere along the line.  I followed both the pre-and - post instructions VERY CAREFULLY.  

It also happens that I have almost this same configuration in another org I support.  It does use the Mass Action credential.  It then chooses a list view from the Opportunity object as the source, which it finds just fine.  Then Action is Process Builder and Flows.  In the working org, 3 invokable PBs show up.  In the new org using 2.4, nothing appears in the drop down even though I do have an active, invokable PB available.

## Expected Behavior
To see the Invokable Process Builder called Payment Reminder Registrations in the dropdown

## Actual Result
No Dropdown

## Source Type

- [ ] Reports
- [x ] List Views
- [ ] SOQL Queries

## Action Type

- [x ] Process Builder & Flows
- [ ] Workflow Rules
- [ ] Email Alerts
- [ ] Quick Actions
- [ ] Apex

## Does your configuration use Named Credentials?
I have tried it both ways, and the result is the same.  

![2021-05-27 MAS 2 1 Action Page](https://user-images.githubusercontent.com/21043974/119883784-8aceb680-bef5-11eb-8661-741b2b9a1f1a.jpg)
![2021-05-27 MAS 2 4 Action Page](https://user-images.githubusercontent.com/21043974/119883786-8b674d00-bef5-11eb-98d0-7bba84c0c315.jpg)
![2021-05-27Process Builder List](https://user-images.githubusercontent.com/21043974/119883787-8b674d00-bef5-11eb-8f96-407fdd379b40.jpg)

**Comment by stargaterx** [comment]
Good afternoon,
I had a similar issue and the only security change I had recently was the source IP was restricted.  Disabling this solved above issue.


![Screen Shot 2021-06-21 at 6 12 28 PM](https://user-images.githubusercontent.com/54185620/122834257-43510580-d2bc-11eb-8924-dd8d01e82afa.png)

**Comment by chicagojcb** [comment]
Hello @stargaterx, thank you for your reply.  I have made the change in session settings but still, no action is avialable when I get to that point in the configuration process.  Do I need to wait?  Log out and log back in, maybe?  

While I was waiting for someone to respond, I tried uninstalling the latest version, and installing ver 2.1, which is working fine in another, similar, small non-profit org.  That didn't seem to matter.  There must be something about this org that is different, but I cannot find it.  This is pro-bono work, but I still feel like I'm wasting everybody's time on this issue.  And yet, this is the only way to do what we're trying to do that I know of that won't cost the org anything for the organization.  

I do appreciate your response!

**Comment by chicagojcb** [comment]
Hello @douglascayers and @stargaterx,

I wanted you to know that I did solve this problem.  I'm not sure how I messed things up, installing and uninstalling, but I went through the setup menu and found all remaining MAS bits in the org and manually deleted them.  Then I installed again, carefully went through the documented setup process, and it is working.  I assume at this point that some sort of user error caused the problem in the first place.  

I am very excited on behalf of this tiny nonprofit client; they will benefit greatly from automating certain kinds of list emails and follow ups.  Thanks again.

Jeanne
